subprojects/docs/src/docs/userguide/declaringDependencies.adoc

```diff
@@ -18,42 +18,53 @@
 Gradle builds can declare dependencies on external binaries, raw files and other Gradle projects. You can find examples for common scenarios in this section. For more information, see the <<dependency_types,full reference on all types of dependencies>>.
 
 [[sec:declaring_binary_dependency]]
-=== Declaring a binary dependency
+=== Declaring a dependency to a module
```

Comment:
:x: We should rename the sample directories as well e.g. `declaringBinaryDependenciesWithConcreteVersion` now that we change the terminology in the text.

Reply:
done